go.mod

```diff
@@ -1,151 +1,135 @@
-module github.com/mongodb-forks/drone-helm3
+module github.com/kanopy-platform/drone-helm3
 
-go 1.21
+go 1.25
 
 require (
 	github.com/golang/mock v1.6.0
-	github.com/helm/helm-2to3 v0.10.1
+	github.com/helm/helm-2to3 v0.11.0
 	github.com/joho/godotenv v1.4.0
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/pkg/errors v0.9.1
-	github.com/stretchr/testify v1.7.0
+	github.com/stretchr/testify v1.10.0
 	gopkg.in/yaml.v2 v2.4.0
-	helm.sh/helm/v3 v3.8.1
-	k8s.io/api v0.23.4
-	k8s.io/apimachinery v0.23.4
-	k8s.io/client-go v0.23.4
+	helm.sh/helm/v3 v3.18.6
+	k8s.io/api v0.33.3
```

Comment:
this was upgraded because of helm.sh/helm v3.18.6 bump

Reply:
If issues are discovered with using the latest version on k8s 1.33; that would be good justification for using an older version that closely matches our environments k8s version.  Its probably unlikely given that we have been using a super older version with no problem.